app/models/episode.rb

```diff
@@ -59,6 +65,10 @@ def enclosure
     enclosures.complete.first
   end
 
+  def image
+    images.complete.first
```

Comment:
note: this is how enclosures and contents for audio both work too, somewhat poorly explained in the readme changes.